<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>cena:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>minimalny wiek:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nast">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>260</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>następny</string>
    </property>
   </widget>
   <widget class="QPushButton" name="poprz">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>poprzedni</string>
    </property>
   </widget>
   <widget class="QLabel" name="pozycja">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>pozycja:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="aktualny">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>30</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="text3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="text4">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>150</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="cena">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="minimalny_wiek">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>null</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>null</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dD">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>300</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>dodaj dom</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dDS">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>300</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>dodaj dom strachu</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dK">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>300</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>dodaj karuzelę</string>
    </property>
   </widget>
   <widget class="QPushButton" name="del">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>340</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>usuń</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ed">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>340</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>edytuj</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nazwa">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>nazwa:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="text5">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>180</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="text6">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>210</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>null</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>null</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="obraz">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>60</y>
      <width>221</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>545</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>dD</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>dodajD()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>348</y>
    </hint>
    <hint type="destinationlabel">
     <x>107</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dDS</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>dodajDS()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>188</x>
     <y>348</y>
    </hint>
    <hint type="destinationlabel">
     <x>170</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dK</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>dodajK()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>261</x>
     <y>348</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nast</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>nastepny()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>324</x>
     <y>308</y>
    </hint>
    <hint type="destinationlabel">
     <x>312</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>poprz</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>poprzedni()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>66</x>
     <y>308</y>
    </hint>
    <hint type="destinationlabel">
     <x>74</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>del</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>usun()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>388</y>
    </hint>
    <hint type="destinationlabel">
     <x>294</x>
     <y>312</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ed</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>edycja()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>138</x>
     <y>385</y>
    </hint>
    <hint type="destinationlabel">
     <x>75</x>
     <y>388</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>wypisz()</slot>
  <slot>dodajD()</slot>
  <slot>dodajDS()</slot>
  <slot>dodajK()</slot>
  <slot>nastepny()</slot>
  <slot>poprzedni()</slot>
  <slot>usun()</slot>
  <slot>edycja()</slot>
  <slot>aktualizuj()</slot>
 </slots>
</ui>
